### ISSUE TITLE
Volume Settings sets volume in GUI but not on actual audio output

### ISSUE THREAD

**Issue by EpicLPer** [willholdoway/hifiberry#24]
Heya!

So this may be a weird behavior, and I'm not even sure if it's the cause of this integration or HiFiBerryOS itself, but when I set the Volume via Home Assistant it only updates it on the Web GUI of HiFiBerryOS, but the actual audio output is unaffected by it. If I use the slider on the Web GUI it works without any issues.

That said, if I use the slider on the Web GUI it doesn't update the slider on Home Assistant, so it seems to only be One-Way so far.

**Comment by willholdoway** [comment]
This is odd @EpicLPer I can't replicate this. Is it sill an issue?

**Comment by EpicLPer** [comment]
Heya, not sure what happened when I tested this back then but it now seems to work again.

However I noticed another issue, when you set a manual minimum volume limit via the web interface (in my case I set it to 70%) and then adjust it via Home Assistant it seems to ignore that low limit. In Spotify and the Web Interface turning the volume down to 50% as example still has audible music playing, where as via the Home Assistant addon it is completely silent as it was before I set that limit. It seems to not respect the low limit and "hard sets" it regardless on the audio DAC.
Not sure if I should open another issue for this.

**Comment by willholdoway** [comment]
Yes I think via the API you can override the limits in HiFi berry OS. you could setup an intermediary slider in home assistant with a constrained range
On 14 Apr 2024 at 12:55 +0100, EpicLPer ***@***.***>, wrote:

Heya, not sure what happened when I tested this back then but it now seems to work again.

However I noticed another issue, when you set a manual minimum volume limit via the web interface (in my case I set it to 70%) and then adjust it via Home Assistant it seems to ignore that low limit. In Spotify and the Web Interface turning the volume down to 50% as example still has audible music playing, where as via the Home Assistant addon it is completely silent as it was before I set that limit. It seems to not respect the low limit and "hard sets" it regardless on the audio DAC.
Not sure if I should open another issue for this.

**Comment by f17mkx** [comment]
Same issue...
